packages/arb-token-bridge-ui/src/hooks/useIsConnectedToOrbitChain.ts

```diff
@@ -0,0 +1,13 @@
+import { useNetwork } from 'wagmi'
+
+import { isNetwork } from '../util/networks'
+
+export function useIsConnectedToOrbitChain() {
```

Comment:
We might want to generalize this hook, to return `isNetwork(chain.id)`, we might have some place where we check if user is currently connected to X or Y

Reply:
I'm not sure what you mean, `chain` will return the currently connected network and `isNetwork` checks if the network is an Orbit network

Reply:
Rather than having `useIsConnectedToOrbitChain`, `useIsConnectedToArbitrum`, we could have 
```
function useIsConnectedTo() {
  const chainId = useChainId()

  return isNetwork(chainId)
}
```

Then we can check `const { isOrbitChain, isArbitrum } = useIsConnectedTo()`

But maybe we don't even need a hook for that, I'm not sure

Reply:
I agree we can refactor it, but also feel that we should not impact logic of already stable `isArbitrum` or `isEthereum` in this PR. We can park this for next improvement.